docs/guides/nrfconnect_examples_configuration.md

```diff
@@ -76,61 +79,55 @@ type. Example on how to use config options was presented below:
+after editing them by typing the following command in the example directory:
 
-        # <board_name> should be replaced with the actual target board name
-        $ west build -b <board_name>
+        # board_name should be replaced with the actual target board name
+        $ west build -b board_name
+
+See Building nRF Connect examples for details.
 
 <hr>
 
 <a name="configuration-structure-overview"></a>
 
 ## Configuration structure overview
 
-Configuration of the application is specified using Kconfig files and typically
-there are many files having impact on the final configuration shape. Actually
-most of the components have their own configuration files, but don't worry,
-there is no need to modify each one of them separately. Below you can see what
-types of files you can meet in the project and which one of them may be
-important from the user's perspective:
-
--   **Software libraries' configuration files**
-
-    As it was previously written many libraries, components and modules have
-    their own configuration files, but in the majority of cases you will be able
-    to override their configuration values in the main application config file,
-    so you will not have to modify their files unless you are interested in
-    developing those libraries.
-
--   **Application configuration file**
-
-    Every example application has its main configuration file called `prj.conf`
-    located in the example directory. It typically contains application specific
-    configuration or most frequently changed options, but actually almost every
-    configuration can be overrided in this file and probably this file is the
-    most important for the User.
-
--   **Overlays**
-
-    Overlay files are usually used to extract configuration for some specific
-    case or feature from the general application configuration. The main
-    difference between them and the application `prj.conf` file is that they are
-    not included automatically, so User may decide whether to build sample with
-    or without them.
-
--   **Board configuration files**
-
-    These are hardware platform dependent configuration files, which are
-    automatically included basing on the compilation target board name and they
-    contain board and its peripherals configuration.
-
-Visit
-[Kconfig](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/kconfig/index.html#kconfig)
-if you are interested in getting more advanced and detailed information about
-used configuration structure.
+The application configuration is specified using Kconfig files.
+Typically, there are many files having impact on the final configuration shape,
+because most of the components have their own configuration files.
+
+There is no need to modify all these files separately.
+See the following list for types of files you can find in the project
+and which of them are important from your perspective:
+
+- **Software libraries' configuration files.**
```

Comment:
I'm not sure which configuration files this section refers to. While building our apps I only get information about three sources:
```
Loaded configuration '/home/damian/src/ncs/zephyr/boards/arm/nrf52840dk_nrf52840/nrf52840dk_nrf52840_defconfig'
Merged configuration '/home/damian/src/chip/config/nrfconnect/sample-defaults.conf'
Merged configuration 'prj.conf'
```

Reply:
Yeah, I guess that I mixed together Kconfig and configuration files. I meant that a lot of libraries (e.g. nrfxlib) have own Kconfig files setting configuration values, by using select or imply. I get your point, as it might be confusing, but on the other hand it is also a part of configuration final shape, so I didn't want to remove it completely. Do you have an idea how to describe it more clearly?

Reply:
Maybe we should start with: "Zephyr RTOS and related software components, like drivers and libraries, provide a set o Kconfig files which define available settings for any application. See an [example](https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/Kconfig.zephyr)"

The final configuration used by an application is a combination of several configuration files:
1. "Board configuration file" - settings required by a given board or default values which can be overridden in the further configuration files... 
2. "Project configuration file(s)" - `prj.conf`, by default, but some CHIP samples use `sample-defaults.conf` and `prj.conf` to extract common settings bla bla
3. "Optional configuration overrides". Can be applied on demand using `west build -b <board_name> -- -DOVERLAY_CONFIG=<overlay_config_file>`

?

Reply:
And I think we shouldn't use "Kconfig file" to describe "*.conf" file.  Let's use configuration files or "*.conf" files maybe?

Reply:
I applied some changes. I think that now the difference between Kconfig and *.conf is clearer and I also added suggested by you information about adding overlays to build.